java/manager-build.xml

```diff
@@ -17,6 +17,10 @@
   <property file="conf/rhn_java.conf"/>
 
   <!-- Other properties -->
+  <property name="container.name" value="uyuni-server" />
```

Comment:
You can remove the `container.name` property: this is automatically discovered by `uyunictl`